### PR TITLE
Add component icons

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 5e21fd3b447e82260c3dd5e5e51e189d21d5086e

https://deploy-preview-18--obs-patterns.netlify.com

File: _config.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Build settings
-markdown: kramdown
+kramdown:
```
Comment (dmarcoux) [inline _config.yml]: This doesn't work somehow, even though it should as described [here](https://jekyllrb.com/docs/configuration/markdown/). Someone has an idea?

Comment (dmarcoux) [reply _config.yml]: I realized that this obviously cannot work as we don't use Kramdown, but [Haml](https://github.com/openSUSE/obs-patterns/blob/9dff0e06c23347397b91e6ff6c267485cbcb50ad/_layouts/post.html#L10)...

**Comment by bgeuken** [comment]
Could you make the colors match with what we use in OBS?

**Comment by dmarcoux** [comment]
@bgeuken Done

**Comment by dmarcoux** [comment]
For now, the icons are all bunched up... I know it's not optimal, but I didn't find a way to have them nicely presented without introducing unrelated tags (like `%br`) in the code example. Any idea?

**Comment by hennevogel** [comment]
> Any idea?

Introduce unrelated tags (like %br) in the code example. This is for other developers, they will understand.

**Review by bgeuken — APPROVED** [review]
LGTM, some more vertical space between the icons would be neat. But I am nitpicking:-)

**Comment by dmarcoux** [comment]
If we start to get a lot of icons, I think that we should change how we present the icons. For now though, it will do

**Pull request merged**